src/platform/plugins/shared/dashboard/server/content_management/v1/cm_services.ts

```diff
@@ -363,9 +387,19 @@ export const dashboardUpdateOptionsSchema = schema.object({
   mergeAttributes: schema.maybe(updateOptionsSchema.mergeAttributes),
 });
 
+export const dashboardCreateResultSchema = schema.object(
+  {
+    data: dashboardAttributesSchema,
+    meta: dashboardResponseMetaSchema,
+  },
+  { unknowns: 'allow' }
+);
```

Comment:
I think `apiError` schema is still useful in something like this. But I've not tested this. Maybe the `schema.never` are too strict?

```suggestion
export const dashboardCreateResultSchema = schema.oneOf([
  schema.object({
    data: dashboardAttributesSchema,
    meta: dashboardResponseMetaSchema,
    error: schema.never(),
  }),
  schema.object({
    data: schema.never(),
    meta: schema.never(),
    error: apiError,
  })
 ]);
```